### ISSUE TITLE
FPS Issue with PS3 Eye on Windows 11

### ISSUE THREAD

**Issue by DrFeelgood66** [opentrack/opentrack#1491]
I have just installed OpenTrack 2022.2.0 on my Windows 11 Pro, AMD Ryzen 3800X Machine, and have spent the last 4 hours on how to get the FPS i set to in PT. By default it's set to 65 FPS, but it shows only 60 FPS. If i set it to 55 FPS i get 50 FPS, and so on down to 30 FPS. Is this normal behavior?

I have installed from Zadig 2.7 libusb0 (v1.2.6.0) many times and also libusbK, but with the same issue, and have removed previously installed drivers from Device Manager where i set it to Show Hidden Devices in the view Devices by driver, and I have tried all my USB ports, and my USB 3 HUB, and the driver is assigned correctly in Device Manager, I have also tried turning on Legacy USB Support in the BIOS. I have followed the guide on the Wiki [https://github.com/opentrack/opentrack/wiki/PS3-Eye-open-driver-instructions](url)

One other thing when calibrating the yellow cross is in my left eye on PS3 Eye view port... shouldn't it been down at my neck?

**Comment by sthalik** [comment]
> By default it's set to 65 FPS, but it shows only 60 FPS. If i set it to 55 FPS i get 50 FPS, and so on down to 30 FPS. Is this normal behavior?

Does the camera selection say "ps3eye open driver"?

> I have installed from Zadig 2.7 libusb0 (v1.2.6.0) many times and also libusbK, but with the same issue

The convoluted procedure is only necessary when the camera fails to start at all.

> One other thing when calibrating the yellow cross is in my left eye on PS3 Eye view port... shouldn't it been down at my neck?

It depends. If the yellow cross doesn't move (or moves only barely) when you pitch and yaw, then it's correct. Otherwise see calibration instructions on the wiki <<https://github.com/opentrack/opentrack/wiki/model-calibration-for-PT-and-Aruco-trackers>>.

**Comment by DrFeelgood66** [comment]
> Does the camera selection say "ps3eye open driver"?
>
Yes it shows PS3 Eye open driver, and it also shows OBS Virtual Camera. 
> 
> The convoluted procedure is only necessary when the camera fails to start at all.
> 
It did fail to start with, when i used Zadig the first couple of times. Got 2 Error messages when i started tracking, something with "Access denied", I don't remember the second error message, but eventually work when i switched to my USB 3.0 Hub.
> 
> It depends. If the yellow cross doesn't move (or moves only barely) when you pitch and yaw, then it's correct. Otherwise see calibration instructions on the wiki <https://github.com/opentrack/opentrack/wiki/model-calibration-for-PT-and-Aruco-trackers>.

Okay I'll take a look
